### ISSUE TITLE
The influence score for Arnoldi under IF is problematic.

### ISSUE THREAD

**Issue by yiliey** [TRAIS-Lab/dattri#184]
The current problem is in test_influence_function.py, where the calculated influence of training set on the test set is Nan in this case.
`# arnoldi
        attributor = IFAttributorArnoldi(
            task=task,
            layer_name=["linear.weight"],
            device=torch.device("cpu"),
            regularization=1e-3,
        )
        attributor.cache(train_loader)
        attributor.attribute(train_loader, test_loader)`

**Comment by tingwl0122** [comment]
Thanks for the issue. Have you tried different regularization coefficients for your use case?

**Comment by TheaperDeng** [comment]
@yiliey Could you have a try on different regularization coefficients later?

**Comment by yiliey** [comment]
Sorry for late reply. I haven't tried on it and  I will have a try later.
